packages/eslint-plugin/tests/rules/restrict-plus-operands.test.ts

```diff
@@ -262,5 +263,25 @@ var foo = pair + pair;
         },
       ],
     },
+    {
+      code: `var foo = 1n + 1`,
+      errors: [
+        {
+          messageId: 'notBigInts',
+          line: 1,
+          column: 11,
+        },
+      ],
+    },
+    {
+      code: `var foo = 1 + 1n`,
+      errors: [
+        {
+          messageId: 'notBigInts',
+          line: 1,
+          column: 11,
+        },
+      ],
+    },
```

Comment:
Can you add tests for `var foo = 1n + "1"` and `var foo = "1" + 1n`?

Reply:
@j-f1 , what do we want to get in these cases, `notBigInts` or `notStrings` errors? 
Because now they are covered by `notStrings` check, that's why I haven't added them.